packages/three-vrm-springbone/src/VRMSpringBoneManager.ts

```diff
@@ -80,135 +89,123 @@ export class VRMSpringBoneManager {
     this.deleteJoint(joint);
   }
 
-  public setInitState(): void {
+  /**
+   * Sorts the joints ensuring they are updated in the correct order taking dependencies into account.
+   */
+  private _sortJoints() {
```

Comment:
[MUST] Put private methods below the public methods. that will also make the diff look better.